osv/impact.py

```diff
@@ -174,11 +185,31 @@ def _get_affected_range(self,
         if equivalent_fix_commit:
           break
 
-      range_collector.add(equivalent_regress_commit, equivalent_fix_commit)
+      # Get the latest equivalent commit in the last_affected range (if
+      # present).
+      equivalent_last_affected_commit = None
+      if last_affected_commits:
+        for last_affected_commit in last_affected_commits:
+          logging.info(
+              'Finding equivalent last_affected commit to %s in %s in %s',
+              last_affected_commit, ref, str(repo_url or 'UNKNOWN_REPO_URL'))
+          equivalent_last_affected_commit = self._get_equivalent_commit(
+              repo,
+              ref,
+              last_affected_commit,
+              detect_cherrypicks=detect_cherrypicks)
+          if equivalent_last_affected_commit:
+            break
+
+        range_collector.add(equivalent_regress_commit, equivalent_fix_commit,
+                            equivalent_last_affected_commit)
 
       if equivalent_fix_commit:
         end_commit = equivalent_fix_commit
         include_end = False
+      elif equivalent_last_affected_commit:
+        end_commit = equivalent_last_affected_commit
+        include_end = False
```

Comment:
`include_end` should be `True` for last_affected

Reply:
Yes, you're right, thanks. Fixed in c38c637